aws-s3/scripts-ccao-data-raw-us-east-1/ccao-condominium_parking.R

```diff
@@ -16,36 +13,37 @@ output_bucket <- file.path(AWS_S3_RAW_BUCKET, "ccao", "condominium")
 
 ##### QUESTIONABLE GARAGE UNITS #####
 
-# Get local file addresses
-source_paths <- c("O:/Condo Worksheets/Condo 2022/")
-
-source_files <- list.files(
-  source_paths,
-  recursive = TRUE, full.names = TRUE
+openxlsx::read.xlsx(
+  "O:/Condo Worksheets/Condo 2022/2022 Desk Review - Questionable Garage Units.xlsx", # styler: off
```

Comment:
**suggestion:** We really shouldn't be ingesting much from the network drives. Can we move these files to the raw data S3 bucket?